nservicebus/sql-persistence/saga.md

```diff
@@ -55,6 +55,36 @@ snippet: SqlPersistenceSaga
 snippet: SqlPersistenceSqlSaga
 
 
+## Requirement for the SqlSagaAttribute
+
+When looking at a standards Saga's `ConfigureHowToFindSaga` it is logical to come to the conclusion that it should be possible to infer the Correlation Id and hence remove the requirement for defining a `[SqlSagaAttribute]`.
```

Comment:
addressed first comment. 

but this

> And then you can close with a statement about defining it twice is duplication and the SqlSaga<> base class was introduced to avoid that duplication.

i am not sure about. throwing stones, in doco, at a flawed API instead of fixing it seems weird